web/src/containers/Login.js

```diff
@@ -27,7 +27,10 @@ class Login extends Component {
     const { username, password } = this.state;
 
     if (authenticated) {
-      return <Redirect to={from} />;
+      if (from.pathname !== '/logout') {
```

Comment:
If we landed on the login page from the logout page, we don't want to redirect back to the logout page.  That'd be a nasty loop!